app/components/Accordion.tsx

```
@@ -0,0 +1,166 @@
+    };
+
+    return (
+        <button
+            onClick={handleClick}
+            className={`
+        w-full px-4 py-3 text-left
+        focus:outline-none
+        transition-colors duration-200 flex items-center justify-between cursor-pointer
+        ${className}
+      `}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)